src/pages/CreateRecover/components/CreateRecoverPassword/components/CreateRecoverPasswodFormFields.tsx

```diff
@@ -53,7 +51,10 @@ export const CreateRecoverPasswordFormFields = ({
               className={classNames(
                 'block w-full p-2 text-sm text-gray-700 placeholder-gray-400 border border-gray-300 rounded relative',
                 {
-                  'border-red-600': showError(field.name)
+                  'border-red-600': getFormHasError({
+                    form: formikProps,
```

Comment:
If this isn't in a loop, this could be a variable, the function is called thrice. 👍